src/antidote_pb_counter.erl

```diff
@@ -60,19 +60,33 @@ encode(Message) ->
 
 %% @doc process/2 callback. Handles an incoming request message.
 process(#fpbincrementreq{key=Key, amount=Amount}, State) ->
-    {ok, _Result} = antidote:append(Key, riak_dt_pncounter, {{increment, Amount}, node()}),
-    {reply, #fpboperationresp{success = true}, State};
+    case antidote:append(Key, riak_dt_pncounter, {{increment, Amount}, node()}) of
+        {ok, _Result} ->
+             {reply, #fpboperationresp{success = true}, State};
+        {error, Result} ->
+            lager:error("Operation failed due to ~p",[Result]),
+            {reply, #fpboperationresp{success = false}, State}
+    end;
 
 %% @doc process/2 callback. Handles an incoming request message.
 process(#fpbdecrementreq{key=Key, amount=Amount}, State) ->
-    {ok, _Result} = antidote:append(Key, riak_dt_pncounter, {{decrement, Amount}, node()}),
-    {reply, #fpboperationresp{success = true}, State};
+    case antidote:append(Key, riak_dt_pncounter, {{decrement, Amount}, node()}) of
+        {ok, _Result} ->
+            {reply, #fpboperationresp{success = true}, State};
+        {error, Result} ->
+            lager:error("Operation failed due to ~p",[Result]),
+            {reply, #fpboperationresp{success = false}, State}
+    end;    
 
-%% @doc process/2 callback. Handles an incoming request message.
 %% @todo accept different types of counters.
 process(#fpbgetcounterreq{key=Key}, State) ->
-    {ok, Result} = antidote:read(Key, riak_dt_pncounter),
-    {reply, #fpbgetcounterresp{value = Result}, State}.
+    case antidote:read(Key, riak_dt_pncounter) of
+        {ok, Result} ->
+            {reply, #fpbgetcounterresp{value = Result}, State};
+        {error, Reason} ->
+            lager:error("Operation failed due to ~p",[Reason]),
```

Comment:
It is not useful the way it is right now!

All that appears is a error that says "Operation failed due to timeout." without any additional information.  Remove it until it's useful or fix it now.